components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/spark/transformations/AnomalyRowsFilter.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.ml.core.spark.transformations;
+
+import org.apache.spark.api.java.function.Function;
+import org.wso2.carbon.ml.core.internal.MLModelConfigurationContext;
+
+/**
+ * A filter to remove normal rows
```

Comment:
It says "AnomalyRowsFilter" ? Same applied to "NormalRowsFilter" too. Contradictory comments :)